presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -70,6 +70,7 @@
 import java.util.Map;
```

Comment:
I have just notice that it would be nicer to move this refactoring commit before "Add null format property to Hive textfile tables".
And then only add `testCreateExternalTableWithNullFormat` in new form.
Now we are adding `testCreateExternalTableWithNullFormat` and then refactoring it in followup commit.

It is nice to have. But not a must do.